[export] Remove torch._export.export

angelayi opened the pull request:
XLA changes: https://github.com/pytorch/xla/pull/6486

Test Plan: CI

Differential Revision: D53316196




cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @fegin @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @voznesenskym @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

ydwu4 reviewed (approved):
Looks great! So glad we're deprecating the old APIs

zhxchen17 reviewed (approved):
expect some codemods

kit1980 commented:
Was the old one widely used?
I can add this to https://github.com/pytorch-labs/torchfix/

jerryzh168 commented on test/test_out_dtype_op.py:
```diff
@@ -121,14 +120,15 @@ def f(x, y):
-            return out_dtype(
-                torch.ops.aten.add_.Tensor, torch.int32, x, y
-            )
+        class M(torch.nn.Module):
```
function no longer supported?

angelayi replied:
Not since https://github.com/pytorch/pytorch/pull/117528

angelayi committed:
[export] Remove torch._export.export (#119095)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/119095

Test Plan: CI

Reviewed By: avikchaudhuri, tugsbayasgalan, ydwu4

Differential Revision: D53316196
